src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.Remote.cs

```diff
@@ -31,6 +31,11 @@ internal abstract partial class AbstractNavigateToSearchService
 
         private static Task<RemoteHostClient.Session> GetRemoteHostSessionAsync(Project project, CancellationToken cancellationToken)
         {
+            if (!RemoteSupportedLanguages.IsSupported(project.Language))
```

Comment:
How does this happen.  AbstractNavitateToSearchService is only instantiated for C# and VB.